src/frontend/src/lib/utils/imageProcessing/index.ts

```diff
@@ -0,0 +1,9 @@
+export const blobFromFile = async () => {
+  let input = <HTMLInputElement>document.getElementById("image");
```

Comment:
toto sa inak vo Svelte robí cez referenciu priamo na element, cez bind (https://svelte.dev/repl/387bf1bbbd134d0193df33ddf712fd40?version=3.14.1), ale asi to môžeš nechať aj takto ak sa ti s tým moc nechce babrať :D

Reply:
Ten link mi nefunguje, ale beru to jako nice to have :)